brave/browser/BUILD.gn

```diff
@@ -25,6 +45,26 @@ source_set("browser") {
     "guest_view/brave_guest_view_manager_delegate.cc",
     "notifications/platform_notification_service_impl.h",
     "notifications/platform_notification_service_impl.cc",
+    "password_manager/brave_password_manager_client.h",
```

Comment:
all //chrome sources should go in chromium_src build files. Please put these in a password manager source set similar to the others in the 2.58.9 branch